ambry-messageformat/src/main/java/com.github.ambry.messageformat/CompositeBlobInfo.java

```diff
@@ -23,9 +26,41 @@
  * total composite blob size, and a list of keys for the blob's data chunks.
  */
 public class CompositeBlobInfo {
+
+  /**
+   * POJO class for holding a store key, the size of the data content it refers to,
+   * and the offset of the data in the larger composite blob
+   */
+  public static class StoreKeyAndSizeAndOffset {
+    private final StoreKey storeKey;
+    private final long offset;
+    private final long size;
+
+    public StoreKeyAndSizeAndOffset(StoreKey storeKey, long offset, long size) {
+      this.storeKey = storeKey;
+      this.offset = offset;
+      this.size = size;
+    }
+
+    public StoreKey getStoreKey() {
+      return storeKey;
+    }
+
+    public long getOffset() {
+      return offset;
+    }
+
+    public long getSize() {
+      return size;
+    }
+  }
+
+
   private final int chunkSize;
```

Comment:
Chunk size would still be used for deserializing V2 metadata content and keys is used so that getKeys() call doesn't have to parse the StoreKeyAndSizeAndOffset object every time it is called